### DISCUSSION TITLE
About DAMASK simulation interruption and restart

### DISCUSSION THREAD

**Discussion by TsingYi11** [damask-multiphysics/DAMASK#39] [Q&A] [answered]
Hi, everyone

I have a question about restarting after interrupting a simulation. I ran the code DAMASK_grid --load tensionY_100.yaml --geom Lath_6323Grain_1081x859x4.vti --material material1.yaml --numerics numerics.yaml, and it was accidentally interrupted at increment 4500. I got a file called Lath_6323Grain_1081x859x4_tensionY_100_material1_numerics.hdf5. Then when I tried to run it with the command DAMASK_grid --load tensionY_100.yaml --geom Lath_6323Grain_1081x859x4.vti --material material1.yaml --numerics numerics.yaml --restart 4500, I got the answer could not open file: Lath_6323Grain_1081x859x4_tensionY_100_material1_numerics_restart.hdf5, this is relatively easy to understand, so I changed my file name to Lath_6323Grain_1081x859x4_tensionY_100_material1_numerics_restart.hdf5. Unfortunately, a new problem occurred, and the error was Note: The following floating-point exceptions are signalling: IEEE_INEXACT_FLAG
ERROR STOP HDF5 error
Error termination. Backtrace:
#0 0x7f1c22a62d4a
#1 0x7f1c22a63859
#2 0x7f1c22a64f87
#3 0x55581cd82fb6
#4 0x55581cd8c51e
#5 0x55581ce5ef3d
#6 0x55581ce5f48f
#7 0x55581cdc42b9
#8 0x55581ce70754
#9 0x55581cd7e10e
#10 0x7f1c225fc082
#11 0x55581cd7e14d
#12 0xffffffffffffffff
This makes me wonder what to do. I sincerely hope to learn a solution to this problem from you.

Kind regards,
Huigao TANG

**Comment by MarDiehl** [comment, marked as answer]
https://github.com/damask-multiphysics/DAMASK/discussions/35

**Reply by TsingYi11** [reply]
Thank you very much for your reply. I have read the relevant content and tried to add f_restart, but I still don’t quite understand it.
Suppose I have calculated the increment to 25, and I want to continue from 20. I added f_restart: 20 at the end of load.yaml, and changed the calculation code to   “DAMASK_grid --load load.yaml --geom Ellipsold_3Grain_16x16x4.vti --material material.yaml --numerics numerics.yaml --restart 20 "   , but there is still the same error. I don't know how to try it. If possible, can you give a specific example? Thank you again for your reply.

**Reply by MarDiehl** [reply]
the instructions on https://damask-multiphysics.org/support/helpdesk.html#problem-description also help here. If you don't provide files, I have to guess what you did and that is too time consuming.
